first-party/jib-quarkus-extension-maven/README.md

```diff
@@ -27,6 +27,9 @@ Note that `<container><mainClass>` should be set to some placeholder value to su
   </dependencies>
 
   <configuration>
+    <!-- set this if you either have quarkus.package.type=fast-jar  -->
+    <!-- or running quarkus version 1.12.0.CR1 or higer (it's default there) -->
+    <fastJarEnabled>true</fastJarEnabled>
```

Comment:
Since `quarkus.package.type` accepts different types, should we name this `<quarkusPackageType>` and default to `fast-jar`? (Initially, we can make the extension return with an error for other types than `fast-jar` or `jar`.

Reply:
Oh, and you should add extension specific options to the extension configuration, not to the main Jib configuration. Here's a [JibLayerFilterExtension example](https://github.com/GoogleContainerTools/jib-extensions/blob/master/first-party/jib-layer-filter-extension-maven/src/main/java/com/google/cloud/tools/jib/maven/extension/layerfilter/Configuration.java) and its [doc](https://github.com/GoogleContainerTools/jib-extensions/tree/master/first-party/jib-layer-filter-extension-maven#examples).